website/scripts/ci/test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -ex
+
+TEST_MODE=$([ "$1" = "code-coverage" ] && echo "test:jenkins" || echo "test")
+# CI=true npm run "${TEST_MODE}"
```

Comment:
commented out because we're not running tests?

Reply:
There is currently no tests to run for the website project so it fails the CI.
They can remove comment  when tests are added